examples/llava/requirements.txt

```diff
@@ -2,3 +2,4 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
 pillow~=10.2.0
 torch~=2.2.1
+torchvision==0.17.1
```

Comment:
This makes the `check-requirements` CI check fail for all new pull-requests with

```
FATAL: Avoid pinning exact package versions. Use '~=' instead.
```

Reply:
thx, I will add this change to the [2.6 PR](https://github.com/ggerganov/llama.cpp/pull/8967)